app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -198,6 +198,21 @@ examples:
         text: "Green"
       - value: "blue"
         text: "Blue"
+  with_custom_heading_level:
+    description: |
+      This allows the heading level to be changed.
+      Heading level will default to `h2` if nothing is passed.
```

Comment:
This is on a new line but yml renders it on the same line as the line above, unless there's a line space. Did you mean for this to be on a separate line?

Reply:
Good shout and nope it doesn't have to be a separate line so I've left it as one line